guides/v2.1/frontend-dev-guide/templates/template-email.md

```diff
@@ -12,81 +12,78 @@ functional_areas:
-6. Customize template content. For details, see <a href="#customize-content">the section on customizing content</a>.
-7. In **Template Styles**, optionally add {% glossarytooltip 6c5cb4e9-9197-46f2-ba79-6147d9bfe66d %}CSS{% endglossarytooltip %} styles for the template. These styles are added inside of a `<style>` tag in the `<head>` of the email. Typically, you'll use the <a href="#email-styles">LESS files</a> to make style changes to emails because some email clients don't support styles in `<style>` tags.
+6. Customize template content. For details, see [the section on customizing content](#customize-content).
+7. In **Template Styles**, optionally add {% glossarytooltip 6c5cb4e9-9197-46f2-ba79-6147d9bfe66d %}CSS{% endglossarytooltip %} styles for the template. These styles are added inside of a `<style>` tag in the `<head>` of the email. Typically, you'll use the [LESS files](#email-styles) to make style changes to emails because some email clients don't support styles in `<style>` tags.
 8. Click **Save Template**.
 9. Now that you have created a template, you must configure that template to be used:
 
     1. If you haven't done so already, log in to the Magento Admin as an administrator.
     2. Click **STORES** > Settings > **Configuration** > SALES > **Sales Emails**.
     3. In the left pane, locate the section that contains the template you want to override. This is the section referenced by **Currently Used For** in your new template. (See step 3 earlier in this section.)
     <br>For example, if you created a "New Order" template, the configuration section is **Order** as the following figure shows.
-    <br><img src="{{ site.baseurl }}/common/images/email_choose-template.png" alt="Choosing a custom template" width="70%" height="70%"/>
+    <br>![Choosing a custom template]({{ site.baseurl }}/common/images/email_choose-template.png){:width="70%"}{height="70%"}
     4. Select your newly created template from the list.
     5. Click **Save Config**.
 
-<h3 id="customize-header-footer">Customize header and footer templates</h3>
+### Customize header and footer templates {#customize-header-footer}
 Every frontend email template includes a header and footer template using these two directives: `{% raw %}{{template config_path="design/email/header_template"}}{% endraw %}` and `{% raw %}{{template config_path="design/email/footer_template"}}{% endraw %}`. By default, those two directives load contents from these files:
  
-  * <a href="{{ site.mage2100url }}app/code/Magento/Email/view/frontend/email/header.html"><code>&lt;Magento_Email_module_dir&gt;/view/frontend/email/header.html</code></a>
-  * <a href="{{ site.mage2100url }}app/code/Magento/Email/view/frontend/email/footer.html"><code>&lt;Magento_Email_module_dir&gt;/view/frontend/email/footer.html</code></a>
+  * [`&lt;Magento_Email_module_dir&gt;/view/frontend/email/header.html`]({{ site.mage2100url }}app/code/Magento/Email/view/frontend/email/header.html)
```

Comment:
Use angle brackets instead of HTML escape characters. The escape characters render as plaintext inside backticks.

> [`<Magento_Email_module_dir>/view/frontend/email/header.html`]({{ site.mage2100url }}app/code/Magento/Email/view/frontend/email/header.html)